STACpopulator/input.py

```diff
@@ -90,7 +104,8 @@ def extract_metadata(self, ds: siphon.catalog.Dataset) -> MutableMapping[str, An
         attrs = xncml.Dataset.from_text(r.content).to_cf_dict()
 
         attrs["access_urls"] = ds.access_urls
-
+        attrs["catalog_url"] = self.catalog_head.catalog_url
```

Comment:
@huard 

For some reason, `ds.access_urls` causes an `AttributeError` on my side. Do you have an idea why that could be?

When I skip this step to avoid the error, my resulting STAC Item JSON (see bottom of output in https://github.com/crim-ca/ncml2stac/pull/3/files#diff-09442021b34927015b7a3703f6ed65611059a807c957feb4dfe2cd9955ccd68e) has the assets references, but the names are different: 
```json
  "assets": {
    "httpserver_service": { ... },
    "opendap_service": { ... },
    ...
  }
```

Seems to be caused by this logic: https://github.com/crim-ca/stac-populator/pull/23/files#diff-ea68b4da6eb5897aa6022852045603d912245daf3f0b044cb9abbd176e758455R196-R199